orchagent/nexthopkey.h

```diff
@@ -13,22 +15,40 @@ struct NextHopKey
 {
     IpAddress           ip_address;     // neighbor IP address
     string              alias;          // incoming interface alias
+    sai_next_hop_type_t nh_type;        // NH type
```

Comment:
The whole issue with force-push is we lost the commit history and I'm not sure on which commit introduced this change.

Reply:
@prsunny Are you saying that the nh_type is an issue now?  This is something that @smaheshm specifically asked for.  I am not sure why having the commit history would be helpful here.

Reply:
this is a big change; we cannot review the entire code for each commit. So individual commits helps in reviewing the incremental code. Not sure why you've to do force-push multiple times